app/validators/valid_keyword_file_validator.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+class ValidKeywordFileValidator < ActiveModel::EachValidator
```

Comment:
How about 
```suggestion
class CSVKeywordFileValidator < ActiveModel::EachValidator
```

Reply:
Fixed in 315cbbc32f5c7012278e5a65246259d9948b0a26